tests/Unit/ApparentHorizons/Test_Tags.cpp

```diff
@@ -30,8 +30,10 @@ namespace {
 // a given average radius and a given center.
 auto create_strahlkorper_y11(const double y11_amplitude, const double radius,
                              const std::array<double, 3>& center) {
-  const size_t l_max = 4, m_max = 4;
+  static const size_t l_max = 4;
+  static const size_t m_max = 4;
 
+  // make clang happy
```

Comment:
remove this comment

Reply:
done

Reply:
Comment cane back...please remove and squash